index.html

```diff
@@ -22,17 +22,329 @@
+            <li class="rect2"><img src="imgformobile/Rectangle2.png" alt="computer"></li>
+        </ul>
+    </nav>
+    <nav class="container16">
+        <ul>
+            <li class="Proj2"><a href="#">Hello-World</a></li>
+        </ul>
+        <nav class="container17">
+            <ul class="data4">
+                <li class="Lang3"><a href="#">HTML/CSS</a></li>
+            </ul>
+        </nav>
+        <nav class="container18">
+            <ul class="data5">
+                <li class="Lang4"><a href="#">Ruby on </a></li>
+            </ul>
+        </nav>
+        <nav class="container19">
+            <ul class="data6">
+                <li class="Lang5"><a href="#">JavaScript</a></li>
+            </ul>
+        </nav>
+    
+        <nav class="container20">
+            <ul class="navlink3">
+                <li class="project2"><a href="#">See this Project</a></li>
+                <li class="arrow1"><a href="#"><img src="imgformobile/Union.png" alt="sign"></a></li>
+            </ul>
+        </nav>
+    </nav>
+    
+    <nav class="container21">
+        <ul class="comp3">
+            <li class="rect3"><img src="imgformobile/Rectangle3.png" alt="computer"></li>
+        </ul>
+    </nav>
+    <nav class="container22">
+        <ul>
+            <li class="Proj3"><a href="#">Hello-World</a></li>
+        </ul>
+        <nav class="container23">
+            <ul class="data7">
+                <li class="Lang6"><a href="#">HTML/CSS</a></li>
+            </ul>
+        </nav>
+        <nav class="container24">
+            <ul class="data8">
+                <li class="Lang7"><a href="#">Ruby on </a></li>
+            </ul>
+        </nav>
+        <nav class="container25">
+            <ul class="data9">
+                <li class="Lang8"><a href="#">JavaScript</a></li>
+            </ul>
+        </nav>
+    
+        <nav class="container26">
+            <ul class="navlink4">
+                <li class="project3"><a href="#">See this Project</a></li>
+                <li class="arrow2"><a href="#"><img src="imgformobile/Union.png" alt="sign"></a></li>
+            </ul>
+        </nav>
+    </nav>
+    
+    <nav class="container27">
+        <ul class="comp4">
+            <li class="rect4"><img src="imgformobile/Rectangle4.png" alt="computer"></li>
+        </ul>
+    </nav>
+    <nav class="container28">
+        <ul>
+            <li class="Proj4"><a href="#">Hello-World</a></li>
+        </ul>
+        <nav class="container29">
+            <ul class="data10">
+                <li class="Lang9"><a href="#">HTML/CSS</a></li>
+            </ul>
+        </nav>
+        <nav class="container30">
+            <ul class="data11">
+                <li class="Lang10"><a href="#">Ruby on </a></li>
+            </ul>
+        </nav>
+        <nav class="container31">
+            <ul class="data12">
+                <li class="Lang11"><a href="#">JavaScript</a></li>
+            </ul>
+        </nav>
```

Comment:
- [ ] The list of project technologies (e.g., "JavasScript", "HTML" and "CSS") are a list ul and each of them a li, please remove all nav tags, all the lists of project should be wrapped in one ul list👍🏿